.travis.yml

```diff
@@ -1,3 +1,5 @@
+sudo: false
```

Comment:
This allows us to use the new docker-based Travis infrastructure, which is faster and allows us to use caching in the future.